docs/gathered-data.md

```diff
@@ -13,1040 +13,1878 @@ archives are packaged as `.tar.gz` files in `/var/lib/insights-operator`.
+both were introduced at `4.3.0` as part of this gatherer and backported to `4.2.10+`.
+- ClusterOperator resources was included at `4.7.0` and backported to `4.6.16+`.
 
 
 ## ClusterProxy
 
-fetches the cluster Proxy - the Proxy with name cluster.
+Collects the cluster Proxy - the Proxy with name cluster.
+
+### API Reference
+- https://github.com/openshift/client-go/blob/master/config/clientset/versioned/typed/config/v1/proxy.go#L30
+- https://docs.openshift.com/container-platform/4.3/rest_api/index.html#proxy-v1-config-openshift-io
+
+### Sample data
+- [docs/insights-archive-sample/config/proxy.json](./insights-archive-sample/config/proxy.json)
+
+### Location in archive
+- `config/proxy.json`
+
+### Config ID
+`clusterconfig/proxies`
+
+### Released version
+- 4.3.0
 
-The Kubernetes api https://github.com/openshift/client-go/blob/master/config/clientset/versioned/typed/config/v1/proxy.go#L30
-Response see https://docs.openshift.com/container-platform/4.3/rest_api/index.html#proxy-v1-config-openshift-io
+### Backported versions
+None
 
-* Location in archive: config/proxy.json
-* See: docs/insights-archive-sample/config/proxy.json
-* Id in config: clusterconfig/proxies
+### Changes
+None
 
 
 ## ClusterVersion
 
-fetches the ClusterVersion (including the cluster ID) with the name 'version' and its resources.
+Collects the ClusterVersion (including the cluster ID) with the name 'version' and its resources.
 
-The Kubernetes api https://github.com/openshift/client-go/blob/master/config/clientset/versioned/typed/config/v1/clusterversion.go#L50
-Response see https://docs.openshift.com/container-platform/4.3/rest_api/index.html#clusterversion-v1config-openshift-io
+### API Reference
+- https://github.com/openshift/client-go/blob/master/config/clientset/versioned/typed/config/v1/clusterversion.go#L50
+- https://docs.openshift.com/container-platform/4.3/rest_api/index.html#clusterversion-v1config-openshift-io
 
-* Location in archive: config/version/
-* See: docs/insights-archive-sample/config/version
-* Location of pods in archive: config/pod/
-* Location of events in archive: events/
-* Location of cluster ID: config/id
-* See: docs/insights-archive-sample/config/id
-* Id in config: clusterconfig/version
+### Sample data
+- [docs/insights-archive-sample/config/version.json](./insights-archive-sample/config/version.json)
+- [docs/insights-archive-sample/config/pod](./insights-archive-sample/config/pod)
+- [docs/insights-archive-sample/events/](./insights-archive-sample/events/)
+- [docs/insights-archive-sample/config/id](./insights-archive-sample/config/id)
+
+### Location in archive
+| Version   | Path														|
+| --------- | ---------------------------------------------------------- |
+| >= 4.2.0  | config/version.json										|
+| >= 4.2.0  | config/id													|
+| >= 4.8.2  | config/pod/openshift-cluster-version/{name}.json			|
+| >= 4.8.2  | events/{namespace}.json									|
```

Comment:
I would assume that:

* the `name` is always `version` and 
* the `namespace` is always `openshift-cluster-version`

If I am correct, it would be nice to have the specific paths here.